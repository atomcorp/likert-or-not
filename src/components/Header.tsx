import * as React from 'react';
import css from '../index.css';

import LikertOrNot from '../index.d';

const Header = (props: LikertOrNot.HeaderProps) => (
  <thead className={css.columns}>{props.children}</thead>
);

export default Header;
