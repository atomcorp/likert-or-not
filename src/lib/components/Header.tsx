import * as React from 'react';
import css from '../Matrix.css';

import {HeaderProps} from '../definitions';

const Header = (props: HeaderProps) => (
  <thead className={css.columns}>{props.children}</thead>
);

export default Header;
