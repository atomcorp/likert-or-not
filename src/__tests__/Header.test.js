import React from 'react';
import Header from '../components/Header';
import renderer from 'react-test-renderer';

test('Header prints text in thead', () => {
  const component = renderer.create(<Header><div>The Header node</div></Header>);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
