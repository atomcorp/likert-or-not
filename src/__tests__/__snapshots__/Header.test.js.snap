// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header prints text in thead 1`] = `
<thead
  className={undefined}
>
  <div>
    The Header node
  </div>
</thead>
`;
